docs/en/advanced-concepts/settings.md

```diff
@@ -43,6 +43,7 @@ Note that the actual commands you would use in your test script differ based on
 |----|----|----|
 |`ignoreUnimportantViews`|Boolean which sets whether Android devices should use `setCompressedLayoutHeirarchy()` which ignores all views which are marked IMPORTANT_FOR_ACCESSIBILITY_NO or IMPORTANT_FOR_ACCESSIBILITY_AUTO (and have been deemed not important by the system), in an attempt to make things less confusing or faster. `false` by default. | `false` or `true` |
 |`allowInvisibleElements`|For UiAutomator2 Driver. Boolean which set whether Android device should show all elements, visible and invisible. `false` by default. | `false` or `true` |
+|`enableNotificationListener`| For UiAutomator2 Driver. Boolean which set whether Android device should enable or disable `NotificationListener`. `true` by default.  | `false` or `true` |
```

Comment:
Typo: Boolean which sets whether the Android device should enable or disable the `NotificationListener`